app/src/main/java/com/mk/tasky/agenda/presentation/detail/event/DetailEventScreen.kt

```diff
@@ -29,6 +29,8 @@ fun DetailEventScreen(
     eventDescription: String,
     onClose: () -> Unit,
     openEditor: (id: String, title: String, body: String, size: Int) -> Unit,
+    openPhotoViewer: (String) -> Unit,
+    deletablePhotoLocation: String?,
```

Comment:
What does this String represent? If it's a Uri, I'd probably represent it as one since a location string could be anything (same for the String arg of `openPhotoViewer`)

Reply:
Changed to Uri
Thanks!